cropper/src/main/java/com/canhub/cropper/CropImageView.java

```diff
@@ -1194,6 +1194,8 @@ private void setBitmap(
       mImageView.clearAnimation();
 
       clearImageInt();
+      mRestoreCropWindowRect = null;
+      mRestoreDegreesRotated = 0;
```

Comment:
Nice!

What you think we add this both setting inside the `clearImageInt()` method? 
Doing this, we can remove
```
      mRestoreCropWindowRect = null;
      mRestoreDegreesRotated = 0;
```
from method `setImageUriAsync(uri)`

Reply:
Yep, makes sense. I applied that suggestion.

I don't have time to look at the sample app, sorry.